06-building-a-program-start-to-finish/writing-tests/output/spec/dplex-test-spec.js

```diff
@@ -0,0 +1,205 @@
+
+describe('InvertedIndex class', () => {
+  beforeEach(() => {
+    this.indexInstance = new InvertedIndex();
```

Comment:
It didn't work. The variable has to be an instance of the context hence the need for the use of `this`